test/configCases/entry/single-entry-point/webpack.config.js

```diff
@@ -0,0 +1,6 @@
+module.exports = {
+	entry: "./single-entry-point",
```

Comment:
For this plugin we want to test the API surface for it. Thankfully this plugin should be able to be used in your config inside of `plugins`. 

Since a bare config requires at least one entry (even with this plugin). How about you create two files. One references an entry point, and then the other one is added via a new SingleEntryPlugin(). You can see an example of this being used in our code in `EntryOptionPlugin` so you know what fields should be what.